src/dvc_objects/fs/system.py

```diff
@@ -36,7 +37,8 @@ def symlink(source: "AnyFSPath", link_name: "AnyFSPath") -> None:
     os.symlink(source, link_name)
 
 
-def _reflink_darwin(src: "AnyFSPath", dst: "AnyFSPath") -> None:
+@functools.lru_cache(maxsize=1)
+def _clonefile():
```

Comment:
Is it worth running a `logger.debug`? Is that expected?

Reply:
I would rather know that we are falling back there, so yeah. This is ran 1 time worst case, so not much reason to remove.

Reply:
yeah, that's true.